### PR TITLE
Prepare the 0.4.2 release

### PR DESCRIPTION
**Pull request by yallop**

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -1,3 +1,8 @@
+0.4.2 (2016-06-01):
+* Add support for ctypes 0.6.0
+* Fix installation bug: the library name is errno_.a
+* Opam: add tags and build-test target
```
Comment (dsheets) [inline CHANGES]: the second two items actually belong with 0.4.1. Sorry :-/

Comment (yallop) [inline CHANGES]: Ah, the 0.4.1 tag moved.  I'll update the entry.

Comment (yallop) [inline CHANGES]: Updated: b608ec2.

**Comment by dsheets** [comment]
Thanks!

**Pull request merged**